src/main/resources/com/google/api/codegen/java/discovery_fragment.snip

```diff
@@ -41,18 +41,27 @@
        params = apiaryConfig.getMethodParams(method.getName)
     public class {@ApiName}Example {
       public static void main(String[] args) throws IOException, GeneralSecurityException {
-        // Authentication is provided by the 'gcloud' tool when running locally
-        // and by built-in service accounts when running on GAE, GCE, or GKE.
-        GoogleCredential credential = GoogleCredential.getApplicationDefault();
-
-        // The createScopedRequired method returns true when running on GAE or a local developer
-        // machine. In that case, the desired scopes must be passed in manually. When the code is
-        // running in GCE, GKE or a Managed VM, the scopes are pulled from the GCE metadata server.
-        // For more information, see
-        // https://developers.google.com/identity/protocols/application-default-credentials
-        if (credential.createScopedRequired()) {
-          credential = credential.createScoped(Collections.singletonList("https://www.googleapis.com/auth/cloud-platform"));
-        }
+
```

Comment:
Does the blank line need to be moved into the `@default` case to not screw up the formatting (and baseline files!) for the ADC case? Or did you intend to insert a blank line there too (which is fine)?

(oh, I see you already changed the baselines below, so that seems intentional. cool)